### DISCUSSION TITLE
"Dont Ask Me Again"

### DISCUSSION THREAD

**Discussion by OrdinarySailor** [stoatchat/discussions#840] [Feature Suggestions]
Feature Request:
It doesnt need to ask me every time. With "dont ask me again" button, i can go to channel without having to click it again every time. 
![Dont Ask Me](https://github.com/user-attachments/assets/0dfd4a44-9029-4c78-b85f-50f6a599b4c2)

**Comment by Tayvick** [comment]
This really should be an option
